mochify/lib/resolve-modules.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const path = require('path');
+
+exports.resolveModules = resolveModules;
+
+function resolveModules(serve, files) {
```

Comment:
Is this necessary? Couldn't the user simply launch the server in the directory that hosts the files?

Reply:
From what I understand it is necessary. E.g. someone runs 

```
mochify ./src/thing.test.js --esm --serve ./src --driver puppeteer
```

which means that without comparing the paths, we'd inject `<script src="./src/thing/test.js"></script>` which would 404 as the server root is already serving from `./src`. It would of course work if we'd just serve from `process.cwd()` in case `--esm` is passed, but then again that could cause a lot of confusion. Not sure, I think I would err on the side of doing the path munging just to be sure.

Reply:
Maybe we could also just disregard `--serve` when `--esm` is being passed, assuming `process.cwd()` as the root and moving on. Would also make it as easy as

```
mochify ./src/thing.test.js --esm --driver puppeteer
```

Reply:
I'm not sure either. Let's get it work first and then decide on how we resolve the paths. It's just a detail.

Reply:
I left it out for now. If we want it back it should be easy to bring back.